File: 240402/DateTime to DateTime/datetime-to-datetime.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        
        if(a < 11)
            System.out.println("-1");
        if(a == 11 && b <11)
            System.out.println("-1");
        if(a == 11 && b == 11 && c <11)
            System.out.println("-1");
        int time = (a-11) * 1440 + (b-11) * 60 + (c-11);
        System.out.println(time);
    }
}

File: 240418/DateTime to DateTime/datetime-to-datetime.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)

    val a = sc.nextInt()
    val b = sc.nextInt()
    val c = sc.nextInt()

    if (b < 11) {
        print(-1)
        return
    }
    if (b == 11 && c < 11) {
        print(-1)
        return
    }

    val result1 = (11 * 24 * 60) + (11 * 60) + 11
    val result2 = (a * 24 * 60) + (b * 60) + c

    print(result2 - result1)
}